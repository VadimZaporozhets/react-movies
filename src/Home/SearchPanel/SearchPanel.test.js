import React from 'react';
import { SearchPanelComponent } from './index';
import { shallow } from 'enzyme';

describe('<SearchPanelComponent />', () => {
    it('should render search panel component', () => {
        const props = {
            classes: {
                marginBottom: {},
                panel: {}
            }
        };

        const wrapper = shallow(<SearchPanelComponent {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});