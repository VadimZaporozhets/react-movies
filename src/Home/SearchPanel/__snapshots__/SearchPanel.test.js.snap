// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchPanelComponent /> should render search panel component 1`] = `
<WithStyles(Grid)
  container={true}
  spacing={40}
>
  <WithStyles(Grid)
    item={true}
    xs={12}
  >
    <WithStyles(Paper)
      elevation={1}
    >
      <WithStyles(SearchInputSubmitComponent)
        onSubmit={[MockFunction]}
        pathname="/"
        searchBy="title"
      />
      <WithStyles(SearchCategoriesComponent)
        handleSearchByChange={[Function]}
        searchBy="title"
      />
    </WithStyles(Paper)>
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
